### PR TITLE
Add statement for explaining the example image

### LINKED ISSUE #43156: Configure Multiple Schedulers
gcr.io/my-gcp-project/my-kube-scheduler:1.0

While Creating the custom scheduler, this image is not available for the general public. If my understanding is correct then my-gcp-project is private and not publically available to eveyone.

### PR DESCRIPTION
**Pull request by Affan-7**
Closes #43156

Deploy preview:

https://deploy-preview-43290--kubernetes-io-main-staging.netlify.app/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#package-the-scheduler

### CODE REVIEW

File: content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md [hunk 1]
```diff
@@ -59,6 +59,9 @@ docker build -t gcr.io/my-gcp-project/my-kube-scheduler:1.0 .
 gcloud docker -- push gcr.io/my-gcp-project/my-kube-scheduler:1.0
 ```
 
+The image name `gcr.io/my-gcp-project/my-kube-scheduler:1.0` is used as an example.
+You need to change it appropriately according to your docker image repository.
```
Comment (Gauravpadam) [inline content/en/docs/tasks/extend-kubernetes/configure-multiple-schedulers.md:63]:
```suggestion
{{< note >}}
The image name `gcr.io/my-gcp-project/my-kube-scheduler:1.0` is used as an example.
You need to change it appropriately according to your docker image repository.
{{< /note >}}
```

**Review by Gauravpadam — COMMENTED** [review]
Just one suggestion, The content of the note is good

**Comment by tengqm** [comment]
@Affan-7 I don't think we need to clarify this. Users should know that this example is not something to try out. To avoid such a misunderstanding, you can just change the image repo to something other than `gcr.io` which is a real image repo.
You can also add a comment to the snippet to remind the users that the image name/tag are just for demo's purpose.

**Pull request merged**